src/main/java/io/swagger/model/media/MediaItem.java

```diff
@@ -50,8 +49,7 @@ public MediaItem name(String name) {
    * Get name
    * @return name
    **/
-  @Schema(example = "The Good Doctor", required = true, description = "")
-      @NotNull
+  @Schema(example = "The Good Doctor", required = false, description = "")
```

Comment:
When adding media types, I updated this validation code to allow you to add a media type by id or by name (a name will do a query against the imdb service to find a match).